.coveragerc

```diff
@@ -1,8 +1,13 @@
 [run]
 branch = True
+omit = nose2/tests/*
 source = nose2
 include = nose2/*
```

Comment:
I think `source` and `include` are mutually exclusive. At least I got a warning at some point. Maybe drop `source` like you did in the report section?

Reply:
Good catch. I'm pretty sure we want `include`, not `source`. I'll double check the coverage docs and update.